### ISSUE TITLE
switch not activated by lever 

### ISSUE THREAD

**Issue by Dickovski** [TeamOpenIndustry/ImmersiveRailroading#1083]
When I place a rail right-hand switch facing south on a raised viaduct placing a lever by the switch fails to cause the switch to work.  I've tried placing the switch along this track in many places several hundred blocks away along the same north-south access and the same thing happens.  Placing a right-hand switch anywhere else and in any orientation works fine, including right next to the existing track.   The viaduct was originally created by laying track at the appropriate height.
 
**System Information**  
 OS: Windows
 System Memory:  "18gb"
 Allocated Memory: "8gb"
Forge-Version: 1.12.1-14.23.5.2847

**Modlist**  
 Upgraded to ImmersiveRailroading-1.12.2-forge-1.9.1 and UniversalModCore-1.12.2-forge-1.1.3 and the same things happen now as with older versions.
 
**latest.log*
[2020-09-19-1.log.gz](https://github.com/TeamOpenIndustry/ImmersiveRailroading/files/7195271/2020-09-19-1.log.gz)
*

**Comment by cam72cam** [comment]
A screenshot would be helpful.  Are you using the golden spike?

**Comment by Dickovski** [comment]
Since I posted this I've laid huge amounts of track with many switches and not come across this problem anywhere else.  I went back to have a look at the first place that I came across the problem, relayed the first switch with this issue, found that I couldn't quite get the same alignment, just a pixel or two out, then tried the lever and it worked.  I prefer to use the position: pixels option and wonder whether the original track was perhaps laid too close to the parallel track.  If I come across the issue again I'll provide a screenshot but at the moment nothing is wrong.  Will probably have to lay the track again in any older sections showing this issue.  I use the golden spike for laying custom curve track, does it have any other uses?
Brilliant mod, I wish someone would come up with a road mod that gave the same smooth rises and curves.
